project.clj

```diff
@@ -56,7 +56,8 @@
                       :dependencies [[org.clojure/clojure "1.8.0-master-SNAPSHOT"]]}
 
              :test-clj {:test-paths ["test/clj"]
-                        :java-source-paths ["test/java"]}
+                        :java-source-paths ["test/java"]
+                        :resource-paths ["test/resources"]}
```

Comment:
Sorry, not related to the PR. When I was going through the code base, I saw that `resouce-test` wasn't actually testing anything -- it was in effect doing something like asserting `(= nil nil)` -- because the `test.txt` resource didn't have its enclosing directory added to the resource path.